src/lib/RequestTypes.ts

```diff
@@ -159,6 +159,8 @@ export interface OnboardingResult {
     walletId: string;
     label: string;
     type: WalletType;
+    hasFile: boolean;
+    hasWords: boolean;
```

Comment:
Well, the Accounts Manager is supposed to be transparent. Right now we are exporting the type of account, but we might not anymore in the future. And these flags are used to signal the user which backups they should do next. And since the Ledger does not have any of these backups, we pretend that we already have them.

However, you are right in saying it's a _bit wrong_, because we also don't even display the backup option in the wallet menu for Ledger accounts, so we are using the type anyway.

Reply:
Yeah, I guess there is no more elegant solution and you're already doing it the best possible way